src/SankeyDiagram.js

```diff
@@ -916,11 +916,10 @@ export default class SankeyDiagram extends React.Component {
       nodes.forEach(n => {
         depthMapXPos[n.depth] = n.x0;
 
-        if (depthMapYPos[n.depth]) {
-          depthMapYPos[n.depth] = Math.min(n.y0, depthMapYPos[n.depth]);
-        } else {
-          depthMapYPos[n.depth] = n.y0;
-        }
+        // For the given depth, set the y equal to the highest positioned y value
+        depthMapYPos[n.depth]
```

Comment:
sorry to be picky, but I think it would be more readable to do 
```
depthMapYPos[n.depth] = depthMapYPos[n.depth] ? Math.min(n.y0, depthMapYPos[n.depth] : n.y0;
```

Reply:
that's true 👍